ql/src/test/results/clientpositive/autoColumnStats_5.q.out

```diff
@@ -202,19 +202,19 @@ PREHOOK: Input: default@partitioned1_n1
 POSTHOOK: query: desc formatted partitioned1_n1 partition(part=1) a
 POSTHOOK: type: DESCTABLE
 POSTHOOK: Input: default@partitioned1_n1
-col_name	data_type	min	max	num_nulls	distinct_count	avg_col_len	max_col_len	num_trues	num_falses	bitvector	comment
```

Comment:
Why was this (duplicated header/column) happening in some cases but in some other cases it is not?